tiatoolbox/dataloader/wsireader.py

```diff
@@ -1077,14 +1096,14 @@ def read_rect(self, location, size, resolution=1.0, units="baseline"):
         )
 
         im_region = self.img[
-            level_location[1]:level_location[1] + baseline_read_size[1],
-            level_location[0]:level_location[0] + baseline_read_size[0],
+            level_location[1] : level_location[1] + baseline_read_size[1],
```

Comment:
DeepSource doesn't like it.

Reply:
Interesting because these lines are not modified by this PR

Reply:
Ah yes, I see what it is. This is black and deepsource style guides conflicting. Style issues are non-blocking on deepsource though.

Reply:
I just merged changed from develop and there were some more non black compliant tests but that is now fixed.